### PR TITLE
I18n: update contributing docs

### PR DESCRIPTION
**Pull request by joshhunt**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by brendamuir — APPROVED** [review]

File: contribute/internationalization.md [hunk 1]
```diff
@@ -88,122 +87,94 @@ See [Lingui Docs](https://lingui.js.org/ref/macro.html#t) for more details.
 Sometimes you may need to translate a string cannot be represented in JSX, such as `placeholder` props. Use the `t` macro for this.
```
Comment (ashharrison90) [inline contribute/internationalization.md:87]: line above this still mentions lingui docs

File: contribute/internationalization.md [hunk 2]
```diff
@@ -88,122 +87,94 @@ See [Lingui Docs](https://lingui.js.org/ref/macro.html#t) for more details.
...
 </Trans>;
 ```
 
 ### React components and HTML tags
 
-Both HTML tags and React components can be included in a phase. The Lingui macro will replace them with placeholder tags for the translators
+Both HTML tags and React components can be included in a phase. The Trans function will handle interpolating it's children properly
 
 ```js
 import { Trans } from "@lingui/macro"
```
Comment (ashharrison90) [inline contribute/internationalization.md:151]: lingui lingui lingui

File: contribute/internationalization.md [hunk 3]
```diff
@@ -73,13 +76,9 @@ For components used all over the site, use just two segments:
...
+### I18next context
 
-In [AppWrapper.tsx](/public/app/AppWrapper.tsx) the app is wrapped with `I18nProvider` from `public/app/core/internationalization/index.tsx` where the Lingui instance is created with the user's preferred locale. This sets the appropriate context and allows any component from `@lingui/macro` to use the translations for the user's preferred locale.
-
-### Message format
-
-Lingui uses the [ICU MessageFormat](https://unicode-org.github.io/icu/userguide/format_parse/messages/) for the phrases in the .po catalogues. ICU has special syntax especially for describing plurals across multiple languages. For more details see the [Lingui docs](https://lingui.js.org/ref/message-format.html).
+We rely on a global i18next singleton (that lives inside the i18next) for storing the i18next config/context.
 
 ### Plain JS usage
```
Comment (ashharrison90) [inline contribute/internationalization.md:83]: think this section could be moved under `How to add a new translation phrase`? e.g. could have 

## How to add a new translation phrase
### JSX
### Plain JS usage

**Review by ashharrison90 — COMMENTED** [review]
looks great! 🙌 

couple of lingui mentions still need removing, other suggestion is just a readability thing but it's up2u 🙂

**Review by ashharrison90 — APPROVED** [review]
perfect 🚀

**Pull request merged**